MCP_Server/server.py

```
@@ -602,6 +603,165 @@ def get_browser_items_at_path(ctx: Context, path: str) -> str:
+    Note: Returns browser items (URIs), not raw filesystem paths.
+    """
+    try:
+        ableton = get_ableton_connection()
+        result = ableton.send_command("get_user_folders")
+        return json.dumps(result, indent=2)
+    except Exception as e:
+        logger.error(f"Error getting user folders: {str(e)}")
+        return f"Error getting user folders: {str(e)}"
+
+@mcp.tool()
+def load_sample(ctx: Context, track_index: int, sample_uri: str) -> str:
```

Ruff (0.14.14)

[warning] Unused function argument: `ctx`

(ARG001)